athena-timestream/src/test/java/com/amazonaws/athena/connectors/timestream/TestUtils.java

```diff
@@ -59,17 +48,17 @@ private TestUtils() {}
 
     private static final String[] AZS = {"us-east-1a", "us-east-1b", "us-east-1c", "us-east-1d"};
 
-    public static QueryResult makeMockQueryResult(Schema schemaForRead, int numRows)
+    public static QueryResponse makeMockQueryResult(Schema schemaForRead, int numRows)
     {
         return makeMockQueryResult(schemaForRead, numRows, 100, true);
     }
 
-    public static QueryResult makeMockQueryResult(Schema schemaForRead, int numRows, int maxDataGenerationRow, boolean isRandomAZ)
+    public static QueryResponse makeMockQueryResult(Schema schemaForRead, int numRows, int maxDataGenerationRow, boolean isRandomAZ)
     {
-        QueryResult mockResult = mock(QueryResult.class);
+        QueryResponse mockResult = mock(QueryResponse.class);
```

Comment:
I approved it to build and it failed, likely because of this. With v2, the request and response objects are final or static I believe so you can't mock them. Instead you just build them and return it. So here, instead of mocking QueryResponse.class, you can just create a QueryResponse object and return it.

Reply:
The build was successful locally, but it failed in GitHub due to a logger line, as indicated by the logs. I have now addressed the issue by adjusting the logs accordingly. Additionally, I have added the Mockito-inline dependency to handle mocking of final classes, ensuring no test failures occur.